ui/src/components/media/calling.rs

```diff
@@ -101,7 +101,7 @@ pub struct ActiveCallProps {
 fn ActiveCallControl(cx: Scope<ActiveCallProps>) -> Element {
```

Comment:
unrelated to your PR but this should be `activate_call_control`.

Reply:
This was done to match the same style as dioxus macro components

Reply:
interesting, do you have a link to some example?

Reply:
If you mean within uplink, then the function [CallUserImageGroup](https://github.com/Satellite-im/Uplink/blob/a7c216be2bf0948fd14ca7688d7c6798caf0431b/ui/src/components/media/calling.rs#L679) is used [here](https://github.com/Satellite-im/Uplink/blob/a7c216be2bf0948fd14ca7688d7c6798caf0431b/ui/src/components/media/calling.rs#L383-L385). In dioxus, you can probably see it more in some of their examples (https://github.com/DioxusLabs/dioxus/blob/b182a6ee75c3f69d5498de3b6d8122b94c8c8fbc/examples/borrowed.rs)

Reply:
Thanks, that's super strange. I've never seen any Rust project make an exception to this convention.